server/controllers/groups.js

```diff
@@ -446,9 +446,21 @@ module.exports = function(app) {
       'isPublic'
     ];
 
+    doUpdate(whitelist, req, res, next);
+  };
+
+  const updateSettings = (req, res, next) => {
+    doUpdate(['settings'], req, res, next);
+  };
+
+  function doUpdate(whitelist, req, res, next) {
     whitelist.forEach((prop) => {
       if (req.required.group[prop]) {
-        req.group[prop] = req.required.group[prop];
+        if (typeof req.group[prop] === 'object') {
+          req.group[prop] = _.defaults(req.required.group[prop], req.group[prop]);
```

Comment:
Is this flipped? According to this: https://lodash.com/docs#defaults, the destination object `req.group[prop]` is the first param.